dae_RelayBoard/FTD2XXLinux.py

```diff
@@ -23,39 +23,53 @@
 import re
 from . import dae_RelayBoard_Common
 from pylibftdi import BitBangDevice, Driver
-            
+
 class FTD2XXLinux(object):
     def __init__(self):
         self.driver = Driver()
         self.bb = None
         self.comport = None
- 
+
     def initialise(self, deviceID, baudRate, mask, bitMode):
         try:
+            found = 0
```

Comment:
Can you please move found and index to the block of code just before they are used.